flang/lib/Optimizer/Transforms/OMPEarlyOutlining.cpp

```diff
@@ -1,6 +1,7 @@
 #include "flang/Optimizer/Dialect/FIRDialect.h"
 #include "flang/Optimizer/Dialect/FIROps.h"
 #include "flang/Optimizer/Dialect/FIRType.h"
+#include "flang/Optimizer/HLFIR/HLFIROps.h"
```

Comment:
Nit: Check whether the pass will need to add HLFIR dialect as a dependency.

Reply:
I checked that it's already listed in the CMake dependencies for all transforms in that directory.